src/module/scene/token-document/document.ts

```diff
@@ -453,7 +453,7 @@ class TokenDocumentPF2e<TParent extends ScenePF2e | null = ScenePF2e | null> ext
             this.object?._onUpdate(changes, {}, game.user.id);
         }
 
-        if (aurasChanged || "width" in changes || "height" in changes) {
+        if ((aurasChanged || "width" in changes || "height" in changes) && this.scene?.isView) {
```

Comment:
I'm not very familiar with the PF2e codebase, so I'm not 100% sure if only calling `checkAuras` on the current scene in view is actually correct. Alternatively we could modify the line below to something like:
```js
this.scene?.checkAuras?.();
```